src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -766,11 +788,20 @@ def __save_log(logs_dir, dir, test_name):
     logger.info(f"[ARGUMENTS] --cache_path={args.cache_path}")
     logger.info(f"[ARGUMENTS] --workers={args.workers}")
     logger.info(f"[ARGUMENTS] --parallel_devices={args.parallel_devices}")
+    logger.info(f"[ARGUMENTS] --split_unit={args.split_unit}")
+    logger.info(f"[ARGUMENTS] --repeat_failed={args.repeat_failed}")
     logger.info(f"[ARGUMENTS] Executable file arguments = {exec_file_args}")
     TaskManager.process_timeout = args.process_timeout
-    conformance = TestParallelRunner(args.exec_file, exec_file_args, args.workers, args.working_dir, args.cache_path, args.parallel_devices)
+    conformance = TestParallelRunner(exec_file_path = args.exec_file,
```

Comment:
`conformance`?? I believe now it is `test_run`:)

Reply:
renamed